styles/Editor.css

```diff
@@ -942,13 +942,16 @@
   vertical-align: super;
   font-size: 75%;
 }
+.ProseMirror .deleted.pending.footnote::before, .ProseMirror .deleted.accepted.footnote::before {
+  text-decoration: line-through;
+}
 
 .ProseMirror .table-footer {
   font-size: 14px;
   padding-top: 8px;
 }
 
-.ProseMirror .table-footnotes-element:not(:first-child)::before {
+.ProseMirror .table-footnotes-element:has(> div):not(:first-child)::before {
```

Comment:
Can you please explain this change?

Reply:
for adding strike-through style for inline footnotes 
![image](https://github.com/Atypon-OpenSource/manuscripts-body-editor/assets/93189633/3ef17363-06d0-4030-b6e7-d90f3d5fc7e0)

Reply:
Enhancement: Improved Selector for better Accuracy

Reply:
I think this response refers to my previous question, not this conversation. right?

Reply:
I have edited the comment

Reply:
Same as my other comment, I'm not comfortable with this level of specificity as it makes it harder to read and manipulate.

Reply:
So it's a workaround for having empty footnotes-element. which is not expected to happen as we expect it to be deleted :/

Reply:
I'm just relocating the line to a another appropriate position, even as we delete the footnote element. This adjustment should be okay.. so it's not a work around 100%